packages/edit-site/src/components/page-patterns/use-patterns.js

```diff
@@ -184,58 +184,19 @@ const selectPatterns = createSelector(
 	]
 );
 
-const patternBlockToPattern = ( patternBlock, categories ) => ( {
```

Comment:
I'm guessing this is the duplication that we're removing. Can you explain a bit where we're using this?

Reply:
It essentially just takes a `wp_block` entity record and maps it to an object that more closely resembles a theme/directory block pattern in order to allow both to be more easily merged into a single list of patterns under the categories in the site editor patterns page and post editor inserter.

Reply:
Don't we have the patterns package now for this kind of things?

Reply:
Yeh, maybe instead of pushing new settings into the block editor we can abstract the duplication out into a shared hook in the patterns package, or something. Will play around with some alternative approaches. Thanks for the feedback.